submissions/TetianaDiachenko/popup_td/index.html

```diff
@@ -0,0 +1,152 @@
+                                        <a href="#">
+                                            <img src="./img/email.png" alt="">
+                                            <p>Mail</p>
+                                        </a> 
+                                    </li>
+                                    <li>
+                                        <a href="#">
+                                            <img src="./img/news.png" alt="">
+                                            <p>News</p> 
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="#">
+                                            <img src="./img/map.png" alt="">
+                                            <p>Map</p> 
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="#">
+                                            <img src="./img/folder.png" alt="">
+                                            <p>Files</p> 
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="#">
+                                            <img src="./img/contact.png" alt="">
+                                            <p>Contacts</p> 
+                                        </a>    
+                                    </li>
+                                    <li>
+                                        <a href="#">
+                                            <img src="./img/camera.png" alt="">
+                                            <p>Photo</p>    
+                                        </a>
+                                    </li>
+                                    <li>
+                                        <a href="#">
+                                            <img src="./img/video.png" alt="">
+                                            <p>Video</p>
+                                        </a> 
+                                    </li>
+                                    <li>
+                                        <a href="#">
+                                            <img src="./img/searchi.png" alt="">
+                                            <p>Search</p> 
+                                        </a>
+                                    </li>                                    
+                                </ul>
+                                <div class="popup__more">
+                                    <input type="checkbox" name="more" id="more" class="more__check">
+                                    <label for="more" class="popup__mbtn">More</label>                                                                                                    
```

Comment:
Logically hide this button when the second part of the popup is open

Reply:
Sorry, but I don't agree with you. I left this button to have a possibility to close the second part of the pop-up.